### ISSUE TITLE
Vies page unable to publish: COVID-19 Vaccination Centre Data Monitor: Accessibility Map  

### ISSUE THREAD

**Issue by uttaraghodke** [inclusive-design/wecount.inclusivedesign.ca#593]
## Describe the bug

We are unable to see a published article under the views page. 

## To Reproduce

1. Go to 'workflow'
2. Click on 'publish'
5. See error
![image](https://user-images.githubusercontent.com/66276957/121551572-c3828b80-c9dd-11eb-89c9-cd8721e5643f.png)

![image](https://user-images.githubusercontent.com/66276957/121551681-d85f1f00-c9dd-11eb-89b9-cb4d677db6c3.png)

**Comment by cindyli** [comment]
The issue is solved by adjusting the permalink value to “/views/mapdemo/” from its original value "mapdemo". On the CMS interface, down below the “permalink” field, there’s an explanation about this format.
